common/nodejs-utils/src/vrs.rs

```diff
@@ -1,6 +1,6 @@
 use node_semver::{Range, Version as NSVersion};
```

Comment:
For context, I ended up on `vrs`, because `clippy::module_name_repetitions` didn't like my initial `version::Version` and I needed something else. 

Honestly, abbreviations seem very common in rust -- half of the standard library uses abbreviated naming: `fs`, `io`, `cmp`, `ptr`, `rc`, `num`, `fmt`, `ops` and I didn't immediately know what all those are, so it feels like there's precedent.